### ISSUE TITLE
Reduce_sum instead of reduce_mean

### ISSUE THREAD

**Issue by Kh4L** [altosaar/variational-autoencoder#7]
Hi @altosaar 

First, thank you for this small tutorial on using `tf.distributions` with a VAE!

```
elbo = tf.reduce_sum(expected_log_likelihood - kl, 0)
```

I wanted to know why you chose to use `reduce_sum` instead of `reduce_mean` when computing the ELBO?

Thanks!

**Comment by jaanaltosaar** [comment]
`reduce_mean` also works - there might be an advantage empirically if the gradients are very large. I chose the sum to match the math which hopefully makes the code easier to understand. The ELBO for the dataset is the sum of the per-datapoint ELBOs.
